common/cmake/gen.cmake

```diff
@@ -116,15 +116,16 @@ function(MUX_GEN)
     DEPENDS
       ${PYTHON3} ${PYTHON3_TARGET}
       ${symbiflow-arch-defs_SOURCE_DIR}/utils/mux_gen.py
-      ${symbiflow-arch-defs_SOURCE_DIR}/vpr/muxes/logic/mux${MUX_GEN_WIDTH}/mux${MUX_GEN_WIDTH}.sim.v
+      #${symbiflow-arch-defs_SOURCE_DIR}/vpr/muxes/logic/mux${MUX_GEN_WIDTH}/mux${MUX_GEN_WIDTH}.sim.v
```

Comment:
What's going here?

Reply:
Previously `mux_gen.py` had a dependency on the stuff under `vpr/muxes/`. I don't think that is needed anymore...

Reply:
So why is the old dependency commented?

Reply:
Done.